scraping/scraping/spiders/amazon_de.py

```diff
@@ -16,8 +16,11 @@ class AmazonSpider(BaseSpider):
     name = TABLE_NAME_SCRAPING_AMAZON_DE
     source, _ = name.rsplit("_", 1)
     allowed_domains = ["amazon.de"]
+    download_delay = 30
 
-    def parse_SERP(self, response: SplashJsonResponse) -> Iterator[SplashRequest]:
+    def parse_SERP(
+        self, response: SplashJsonResponse, user_agent: Optional[str] = None
+    ) -> Iterator[SplashRequest]:
```

Comment:
I know it is possible to pass arguments. However, I mixed it up with `parse_PRODUCT` and meant that it is implemented in the Base class. But I still prefer using a middleware, it feels more nativ to use their feature instead of implement our own, since it does basically the same?